### ISSUE TITLE
Wrong inlay hint when passing *arg or **kwarg

### ISSUE THREAD

**Issue by fedyakov** [microsoft/pylance-release#4930]
When passing arguments to a function as *ars or **kwargs, an inlay hint with the name of just the first parameter is shown. I think, it either must be no hint or all the parameter names that actually match, if it's possible.

## Environment data

-   Language Server version: 2023.10.10

## Code Snippet

```python
def f(a, b):
    return a + b

args = [1, 2]
f(*args)

kwargs = {"a": 1, "b": 2}
f(**kwargs)
```

<img width="312" alt="image" src="https://github.com/microsoft/pylance-release/assets/1201217/07bbf7cf-4df6-4ec7-9b5c-2338e1b912d5">

**Comment by rchiodo** [comment]
I believe @bschnurr you fixed this to pass no hint? (Validating release now)

**Comment by heejaechang** [comment]
This issue has been fixed in prerelease version 2023.10.51, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#20231051-26-October-2023-PreRelease)
